model_compression_toolkit/core/common/quantization/quantization_params_generation/symmetric_selection.py

```diff
@@ -52,8 +52,14 @@ def symmetric_selection_tensor(tensor_data: np.ndarray,
         Optimal threshold to quantize the tensor in a symmetric manner.
     """
 
-    unsigned_tensor_data = np.abs(tensor_data)
-    tensor_max = get_tensor_max(unsigned_tensor_data, per_channel, channel_axis)
+    if per_channel:
```

Comment:
This part is almost duplicated from "get_tensor_max".
Is it possible to update the function itself and continue to use it instead of duplicating most of the code? (or create a "get_symmetric_tensor_max" that uses the "get_tensor_max" function)